automation/routines_test.go

```diff
@@ -87,66 +87,66 @@ func TestRoutinesSubRoutines(t *testing.T) {
 	t.Run("wildcard", test(testCase{
 		patterns: []string{"*"},
 		expectLines: []string{
-			`level=INFO msg=OK routine=folder1/folder1/routine1`,
-			`level=INFO msg=OK routine=folder1/folder1/routine2`,
-			`level=INFO msg=OK routine=folder1/folder2/routine1`,
-			`level=INFO msg=OK routine=folder1/folder2/routine2`,
-			`level=INFO msg=OK routine=folder2/folder1/routine1`,
-			`level=INFO msg=OK routine=folder2/folder1/routine2`,
-			`level=INFO msg=OK routine=routine1`,
-			`level=INFO msg=OK routine=routine2`,
+			`level=INFO msg=OK app=github.com/clarify/clarify-go routine=folder1/folder1/routine1`,
```

Comment:
Not sure why these tests failed. Hasn't been any changes in the repo since February, and they ran fine then.